src/app/shared/components/capture-receipt/camera-preview/camera-preview.component.ts

```
@@ -91,22 +89,22 @@ export class CameraPreviewComponent implements OnInit, OnChanges {
         this.getFlashModes();
       });
     }
   }
 
-  stopCamera() {
+  stopCamera(): void {
     //Stop camera only if it is in RUNNING state
     if (this.cameraState === CameraState.RUNNING) {
       this.cameraState = CameraState.STOPPING;
-      from(this.cameraPreviewService.stop()).subscribe((_) => (this.cameraState = CameraState.STOPPED));
+      from(this.cameraPreviewService.stop()).subscribe(() => (this.cameraState = CameraState.STOPPED));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)